source/api/content-state/0.9/index.md

```diff
@@ -0,0 +1,713 @@
+* _load Manifest M, navigate such that Range R is selected, and start playing the time-based canvas C within Range R at time t=T_
+
+These intentions can be expressed formally as an Annotation that targets the intended resource, or part of the resource. This content state Annotation must be easily _passed_ into web applications, for example, as a query string parameter, or an HTML `data-` attribute.
+
+### 2.1. Annotation Model for Content State
+
+The target of the Annotation is the IIIF resource, using exactly the same patterns as any other IIIF-targeting Annotation that a IIIF viewer might encounter.
+
+The target could be any resource described by the Presentation API, for example, a:
+
+* Manifest
+* Range
+* Canvas
+* spatial or temporal fragment of a Canvas
+* spatial or temporal point on a Canvas
+
+The Annotation must contain enough information about de-referenceable resources to show the content in context. For example, a Canvas is often not enough information for a viewer to show the intended view; the Manifest the Canvas is part of needs to be declared so that the client can load that Manifest first, and then find the Canvas within it.
+
+### 2.2 Form of Annotation
+
+Any Annotation may have one or more [motivations][org-w3c-webanno-motivation], that provide the reason(s) why is was created. For example, `bookmarking`. 
+
+A content state Annotation always has the motivation `contentState`. This motivation is not defined by either the [W3C Web Annotation Data Model][org-w3c-webanno] or the IIIF Presentation API, and is chosen to avoid potential ambiguity when the target of the content state Annotation is itself an Annotation. The content state annotation may also have additional motivations such as `bookmarking`, `identifying` and so on, as defined by the W3C Web Annotation Model, but it is its particular `contentState` motivation that produces the required behavior in compatible software.
+
+A content state annotation can take several formats. All of the four examples below are equivalent to the following annotation, which is the simplest content state: a reference to a IIIF Manifest.
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "https://example.org/Annotation-server/bookmarks/b1",
+  "type": "Annotation",
+  "motivation": ["contentState"],
+  "target": {
+     "id": "https://example.org/iiif/item1/manifest",
+     "type": "Manifest"
+  }
+}
+```
+
+Publishers _SHOULD_ provide the content state annotation in one of the following forms.
+A client _SHOULD_ be able to accept and process the content state in all of these forms. 
+
+#### 2.2.1 Full Annotation
+
+The content state _MAY_ be supplied as JSON-LD, as a full Annotation with the motivation `contentState`, as in the example above.
+
+The target of the annotation is in this case a complete IIIF resource (here, a Manifest) but in more complex cases, the target could be a part of a IIIF resource.  
```

Comment:
Suggest adding parenthetical commas:  ... annotation is, in this case, a ...

Reply:
added